keras_cv/datasets/waymo/load.py

```diff
@@ -67,6 +72,7 @@ def simple_transformer(frame):
     load("/path/to/tfrecords", simple_transformer, output_signature)
     ```
     """
+    assert_waymo_opendata_installed("keras_cv.datasets.waymo.load()")
```

Comment:
This is the only place where you include parens in the symbol name -- let's either exclude it here or include it everywhere

Reply:
Included everywhere